ddtrace/internal/processor/stats.py

```diff
@@ -0,0 +1,243 @@
+from collections import defaultdict
+import typing
+
+import ddtrace
+from ddtrace import config
+from ddtrace.vendor.ddsketch.ddsketch import LogCollapsingLowestDenseDDSketch
+from ddtrace.vendor.ddsketch.pb.proto import DDSketchProto
+
+from . import SpanProcessor
+from ...constants import SPAN_MEASURED_KEY
+from .._encoding import packb
+from ..agent import get_connection
+from ..compat import get_connection_response
+from ..compat import time_ns
+from ..forksafe import Lock
+from ..hostname import get_hostname
+from ..logger import get_logger
+from ..periodic import PeriodicService
+from ..writer import _human_size
+
+
+if typing.TYPE_CHECKING:
+    from typing import DefaultDict
+    from typing import Dict
+    from typing import List
+    from typing import Optional
+
+    from ddtrace import Span
+
+    from ..agent import ConnectionType
+
+
+log = get_logger(__name__)
+
+
+def _is_top_level(span):
+    # type: (Span) -> bool
+    """Return whether the span is a "top level" span."""
```

Comment:
can we expand this description to explain what we are even checking here?

Reply:
I held off since this is already done in the existing top level stuff (which we'll merge this logic with)